webui/react/src/pages/ExperimentDetails/ExperimentVisualization.tsx

```diff
@@ -283,7 +283,7 @@ const ExperimentVisualization: React.FC<Props> = ({
       type="warning" />;
   } else if (pageError) {
     return <Message title={PAGE_ERROR_MESSAGES[pageError]} type={MessageType.Alert} />;
-  } else if (!hasLoaded) {
+  } else if (!hasLoaded && experiment.state !== RunState.Paused) {
```

Comment:
**chore:** this looks great. I think there is another bug up near where `hasLoaded` is being set. Can you update this too please 🙏 
```
hasLoaded: batches && setMetrics,
```
should be this instead:
```
hasLoaded: batches && metrics,
```